boardgame-ui/src/pages/TimerCreationPage.vue

```diff
@@ -1,265 +1,36 @@
 <template>
   <div>
     <HeroTitlePageLayout :title="$t('timer.add-edit.title')"/>
```

Comment:
We have "Edit timer" in hero title, then "Create timer" in page title, a bit strange

![image](https://user-images.githubusercontent.com/36953600/57982152-b8004880-7a41-11e9-9290-162c23745b52.png)

Reply:
👍  (will be removed for this release though)